### ISSUE TITLE
Clarify relationship with elm-units etc.

### ISSUE THREAD

**Issue by ianmackenzie** [ianmackenzie/elm-geometry#108]
Do a pass through the docs and point out where types/modules come from other packages; may also be worth adding a section to the README like "Relationship to other packages" or similar.

**Comment by ianmackenzie** [comment]
See also https://github.com/ianmackenzie/elm-geometry-svg/issues/12, https://github.com/ianmackenzie/elm-3d-scene/issues/28

**Comment by dimpase** [comment]
Note that the  "Using the package" instructions in https://package.elm-lang.org/packages/ianmackenzie/elm-geometry/latest/ don't work out of the box, as `elm-units` is not in direct dependencies after `elm install ianmackenzie/elm-geometry` mentioned just before that section.  

A clarification would be great, as this is perhaps the first code using `elm-geometry` people might try (and get an error due to a missing dependency).

**Comment by ianmackenzie** [comment]
Good point! I've updated the README so that at least that is clear, but there probably still needs to be a more thorough review at some point.
